### ISSUE TITLE
bug: blogpost is very slow when you have 5k Members

### ISSUE THREAD

**Issue by sunnysideup** [silverstripe/silverstripe-blog#640]
We had a site with 5k Members and the CMS kept running out of memory when we load a blog post.

**Comment by michalkleiner** [comment]
This has been addressed in past IIRC, might be a regression that came back? There was a way how not to load all the members until interacting with the member dropdown. Unfortunately, I don't remember much more.

**Comment by sunnysideup** [comment]
yes, I think you can change the set up in Blog to avoid this, but the thing is that this you have no idea why it loads so slow / trips over ... I'd say that the way it should be changed is that only members who can access the CMS can be authors ... I mean, this idea that any member can write a blog post is lovely, but feels more like and edge case (e.g. a members site where an author for a blog is just an ordinary members and somehow needs to be linked to blog that has been created and published by a CMS editor). 

here is my pull request: https://github.com/silverstripe/silverstripe-blog/pull/641
